db/seeds/case_stages.rb

```diff
@@ -84,19 +84,10 @@
 )
 
 SeedHelper.update_or_create_case_stage!(
-  id:9,
-  description: 'Pre PTPH (evidence served)',
-  unique_code: 'NOPTPHWPPE',
-  position: 90,
-  case_type_id: CaseType.find_by(fee_type_code: 'GRGLT').id,
-  roles: %w(lgfs)
-)
-
-SeedHelper.update_or_create_case_stage!(
-  id: 10,
-  description: 'Pre PTPH (no evidence served)',
-  unique_code: 'NOPTPHNOPPE',
-  position: 100,
-  case_type_id: CaseType.find_by(fee_type_code: 'GRDIS').id,
+  id: 12,
+  description: 'Pre PTPH or PTPH adjourned',
+  unique_code: 'PREPTPHADJ',
+  position: 110,
+  case_type_id: CaseType.find_by(fee_type_code: 'GRRAK').id,
```

Comment:
treating as a cracked trial 'GRRAK' here - is that right?

Reply:
saw that business regs snippet 👍

Reply:
I think we should leave the old types/seeds in there because they exist in the database and their are claims that are using them so we can't get rid of just yet.

we should do it later. can chat about this as they will need to be excluded from the list of stages tho